NVDSuppressions.xml

```diff
@@ -85,4 +85,12 @@
 		</notes>
 		<cve>CVE-2022-39349</cve>
 	</suppress>
+	<suppress>
+		<notes>
+			<![CDATA[
+   file name: Microsoft.JScript.dll
```

Comment:
There should be some notes detailing why this suppression exits (likely a false positive based on the CVE being for a JavaScript thing???)

Reply:
We regularly upgrade packages and go through suppressions to remove unnecessary entries. 
I will add notes and create user story to go through once upgraded to .net 8.